fluent/client/ws_client.go

```diff
@@ -112,7 +115,14 @@ func (wcf *DefaultWSConnectionFactory) New() (ext.Conn, error) {
 
 	conn, resp, err := dialer.Dial(wcf.URL, header)
 	if resp != nil && resp.Body != nil {
-		// TODO: dump response, which is second return value from Dial
+		bodyBytes, readErr := io.ReadAll(resp.Body)
+		if readErr == nil {
+			bodyString := string(bodyBytes)
+			if resp.StatusCode == 403 {
```

Comment:
I don't think we should only do this for 403's, I think we should do this if StatusCode != nil

Reply:
this issue has been addressed , @danielgrist request you to review it again

Reply:
@danielgrist what do you mean `StatusCode != nil` - `StatusCode` is an integer type, the zero value is 0, not `nil`.